python/pfs/drp/stella/extinctionCurve.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+from scipy import interpolate
+
+__all__ = ("ExtinctionCurve", "F99ExtinctionCurve")
+
+
+class ExtinctionCurve:
+    """Dust extinction curve.
```

Comment:
Point out that this is an abstract base class.
Maybe you should inherit from `abc.ABC` and make `_extinction` an `@abstractmethod`?